pkg/k8s/apps/translate.go

```diff
@@ -260,14 +262,13 @@ func TranslateResources(c *apiv1.Container, r model.ResourceRequirements) {
 		c.Resources.Requests[apiv1.ResourceEphemeralStorage] = v
 	}
 
-	if v, ok := r.Requests[model.ResourceAMDGPU]; ok {
-		c.Resources.Requests[model.ResourceAMDGPU] = v
-	}
-
-	if v, ok := r.Requests[model.ResourceNVIDIAGPU]; ok {
-		c.Resources.Requests[model.ResourceNVIDIAGPU] = v
+	for resname, v := range r.Requests {
+		if strings.Contains(string(resname), model.ResourceNamespaceSeparator) {
```

Comment:
Let's move the constant to the top of the file, I think it'd be better since we are just using it here. WDYT?

Reply:
How 'bout we reduce the duplication by a simple loop; now it's not even used more than once,  and it doesn't need to be a const at all?